src/test/java/bio/terra/common/ValidationUtilsTest.java

```diff
@@ -76,4 +88,64 @@ public void testValidationOfValidInputString() {
     // no exception is thrown
     ValidationUtils.requireNotBlank("abc", "error msg");
   }
+
+  @Test
+  public void testRelationshipValidationDifferentDataType() {
+    RelationshipTermModel fromTerm = new RelationshipTermModel().column("hasCat").table("person");
+    RelationshipTermModel toTerm = new RelationshipTermModel().column("ownerId").table("car");
+    defineSampleTables();
+
+    LinkedHashMap errors =
+        ValidationUtils.validateMatchingColumnDataTypes(fromTerm, toTerm, tables);
+    assertThat("There should be one error.", errors.size(), equalTo(1));
+  }
+
+  @Test
+  public void testRelationshipValidationSameDataType() {
+    RelationshipTermModel fromTerm = new RelationshipTermModel().column("id").table("person");
+    RelationshipTermModel toTerm = new RelationshipTermModel().column("ownerId").table("car");
+    defineSampleTables();
+
+    LinkedHashMap errors =
+        ValidationUtils.validateMatchingColumnDataTypes(fromTerm, toTerm, tables);
+    assertThat("The data types match so there should not be an error.", errors.size(), equalTo(0));
```

Comment:
This could be a good place to use the `empty` matcher:
```
assertThat("The data types match so there should not be an error.", errors, empty());
```

Reply:
Oh nice, I didn't know about that matcher option.  However, it looks like it doesn't work with a map without converting to a List. This seems like unnecessary overhead in this case.